index.html

```diff
@@ -17,27 +17,33 @@
         <div id="main">
             <div id="left">
                 <img src="pics/openageinverse.svg" alt="openage" id="logo">
-                <h1>A free (as in <i>freedom</i>) clone of the Age of Empires II engine</h1>
+                <h1>Free cross-platform RTS game engine based on Age of Empires mechanics</h1>
                 <div id="mainbuttons">
-                    <a href="https://github.com/SFTtech/openage" class="code" id="button">View the Code</a>
-                    <a href="https://github.com/SFTtech/openage/tarball/master" class="tgz" id="button">Download .tar.gz</a>
+                    <a href="https://repo.openage.dev" class="code" id="button">Repository</a>
+                    <a href="https://bintray.com/simonsan/openage-packages/openage-windows-nightly/_latestVersion" class="tgz" id="button">Windows Packages</a>
+                    <a href="https://repology.org/project/openage/versions" class="tgz" id="button">Linux Packages</a>
                 </div>
             </div>
             <div id="rightcontainer"><div id="right">
                 <p>You know Age of Empires. You know its bugs. You know the fun playing it.</p>
-                <p>We are creating a free (as in freedom) clone of the Age of Empires II engine in C++17 from scratch (GPLv3+). We use "modern" technologies like OpenGL/GLSL, Python, Qt5, CMake, ...</p>
-                <p>We aim to create an exact clone (for now), using the original media files.</p>
-                <p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively, minus features, plus additional epic bugs (or was it vice versa?).</p>
-                <p>The project is run by volunteers: Students and professionals from all over the world.</p>
-                <p>Join us and help to create an awesome game!</p>
+				<p>We are creating a free (as in freedom, GPLv3+) cross-platform RTS game engine based on Age of Empires mechanics in C++17. We use "modern" technologies like OpenGL/GLSL, Python, Qt5, CMake, ...</p>
+				<p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively.</p>
+				<p>We aim to create a game engine that provides the original look and feel. Until we have free replacement assets, we can convert original media files provided by the user to an format compatible to openage.</p>
+
+				<p><b>A look into the future:</b> openage has a lot of improvements planned. It is going to add completely new possibilities and will provide better performance due to different design decisions being made from the beginning on, e.g. being multithreaded and using modern technologies like Vulkan rendering. Our self-designed entity-component-system <a href="https://github.com/SFTtech/nyan/">nyan</a> will help us exposing a Modding API making openage more easy for modders to use features of openage in their mods.</p>
```

Comment:
We'll see about better performance :D I would not claim this as of yet, since so much can change until the release.